src/components/shared/DetailsWidget/RolesDetail/RolesDetail.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { ActionUserRoles } from '~utils/colonyActions';
```

Comment:
This looks like a type, is there a good reason for having it in utils? Also the types file is called `actions`, whereas utils are `colonyActions` - we should choose one and stick to it